### PR TITLE
Translation-ja: Add trusted-committer/ja/01-introduction-ja

### PR DESCRIPTION
**Pull request by ystk**
This is the first PR for #367 .

### CODE REVIEW

**Comment by lenucksi** [comment]
Hey @ystk! Thanks a lot for translating more of the learing path - I'm curious to see what the article that I was a bit more active on writing than with the others will look like 😄 🎉 
As always: I'm happy to merge once we have another review from another Japanese-speaking person. Could we maybe interest you for this again, @johtani?

File: trusted-committer/ja/01-introduction-ja.asciidoc [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+POは、Trusted Committerと頻繁に対話します(例えば、要求された機能や提供された機能が実際に製品に属することを確認します)。
+特に、小規模で草の根のInnerSourceコミュニティでは、Trusted Committerは通常POとしても活動します。
+より詳細については、 https://innersourcecommons.org/resources/learningpath/product-owner/index[ラーニングパスのプロダクトオーナーセグメント] を確認してください。
+
+=== 役割の名前が重要な理由
+
+Trusted Committerの役割は、成功したすべてのInnerSourceコミュニティにあるが、すべてのコミュニティがその名前を使用しているわけではありません。
+いくつかのコミュニティでは、Maintainer（メンテナー）という用語を使用していますが、この用語は他の技術的な役割、例えばGitHubで定義されている「Maintainer」の役割と競合するものです。
+Apacheも同様に _コミッター（Committer）_ という用語を使用しますが、その役割に関連付けられる責任の数は少なく、ほとんどが技術指向の責任です。コミュニティ指向の責任が追加されたTrusted Committerの役割は、それを超えています。
+Trusted Committerの「信頼された（Trusted）」とは、その人が信頼されていることを意味しており、したがって、経営陣とコミュニティの両方から、彼らの仕事をする権限を与えられています。
```
Comment (johtani) [inline trusted-committer/ja/01-introduction-ja.asciidoc:40]: 「経営陣」という言葉が少ししっくりこなかったのですが、代わりのいい言葉も浮かんでこなかったです。introductionの02では「マネージメント層」という役だったので「マネージメント」でもいいのかもしれません。

**Review by johtani — COMMENTED** [review]
Sorry for the late reply. 
I left some comments.

**Comment by rrrutledge** [comment]
@johtani you're doing a great service in these reviews!

**Comment by ystk** [comment]
@johtani Thanks for your suggestions! I updated the file. Please check it.

**Review by johtani — APPROVED** [review]
LGTM!

**Review by rrrutledge — APPROVED** [review]
Woo hoo!  Thank you!

**Pull request merged**